# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/avm/xautomation/png_h-location.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN xautomation-0.96/image.c xautomation-0.96-new/image.c
--- xautomation-0.96/image.c	2003-12-15 19:44:01.000000000 +0100
+++ xautomation-0.96-new/image.c	2004-12-06 19:45:46.798031352 +0100
@@ -22,7 +22,7 @@
 #include <stdlib.h>
 
 #include <X11/XWDFile.h>
-#include <png.h>
+#include <libpng/png.h>
 
 #include "image.h"
 #include "debug.h"
